src/library_pthread.js

```diff
@@ -1094,8 +1135,10 @@ var LibraryPThread = {
   _emscripten_notify_task_queue: function(targetThreadId, currThreadId, mainThreadId, queue) {
     if (targetThreadId == currThreadId) {
       setTimeout(() => executeNotifiedProxyingQueue(queue));
+    } else if (targetThreadId == mainThreadId) {
+      postMessage({'cmd' : 'processProxyingQueue', 'queue' : queue});
```

Comment:
Why is the main thread special here? (Is this just an optimization?)

Reply:
Yeah, in principle the main thread could listen only on the message broker's channel for messages from all other threads, but since it already listens directly to every thread and the code isn't that complex, I figured it would be better to keep messages to the main thread as 1 hop instead of 2.

Reply:
Perhaps worth a comment?